test/exfmt/integration/map_test.exs

```diff
@@ -47,11 +47,11 @@ defmodule Exfmt.Integration.MapTest do
   end
 
   test "chained map get" do
-    assert_format "map.key.another.a_third\n"
+    "map.key.another.a_third" ~> "map.key().another().a_third()\n"
```

Comment:
It's a simple change to avoid adding parentheses to zero arity non-qualified calls, so we'll be fine whatever the guidance.